src/OSPSuite.InstallationValidator.Core/Reporting/ValidationStateTeXBuilder.cs

```diff
@@ -23,21 +23,21 @@ public ValidationStateTeXBuilder(ITeXBuilderRepository builderRepository)
       public override void Build(ValidationState validationState, OSPSuiteTracker buildTracker)
       {
          var color = colorFor(validationState);
-         _builderRepository.Report(new object[] { "Result of the Validation: ", new ColorText($"{validationState}", color), Environment.NewLine}, buildTracker);
+         _builderRepository.Report(new object[] {"Result of the Validation: ", new ColorText($"{validationState}", color), Environment.NewLine}, buildTracker);
       }
 
       private Color colorFor(ValidationState validationState)
       {
          switch (validationState)
          {
-               case ValidationState.Invalid:
-                  return Color.Red;
-               case ValidationState.ValidWithWarnings:
-                  return Color.Orange;
-               case ValidationState.Valid:
-                  return Color.Green;
-               default:
-                  return Color.Empty;
+            case ValidationState.Invalid:
```

Comment:
That should be removed probably because we are displaying color somewhere else as well